### PR TITLE
Update Nginx 1.12

### PR DESCRIPTION
**Pull request by butuh**
#200 #168 
update nginx to 1.12.0-r1
image build:
a

### CODE REVIEW

**Comment by djdefi** [comment]
Thanks for the PR! It looks like we may need to fiddle with the configured repos to get the build to work.

**Comment by butuh** [comment]
I dont use travist to build this... sorry... using local docker.. will try aggain...

**Comment by butuh** [comment]
just update image from alpine to nginx:1.3.0-alpine

File: Dockerfile [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
 MAINTAINER Alt Three <support@alt-three.com>
 
 EXPOSE 8000
 CMD ["/sbin/entrypoint.sh"]
-
 ARG cachet_ver
 ENV cachet_ver ${cachet_ver:-master}
 
-ENV NGINX_VERSION 1.10.3-r0
+#ENV NGINX_VERSION 1.12.0-r1
```
Comment (djdefi) [inline Dockerfile]: Can go ahead and remove these lines fully, no need to comment them out -- we have `git` for history if we need this again :)

File: Dockerfile [hunk 2]
```diff
@@ -1,24 +1,22 @@
...
 ENV COMPOSER_VERSION 1.4.1
 
-RUN echo "@edge http://nl.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories
+#RUN echo "@edge http://nl.alpinelinux.org/alpine/edge/main" >> /etc/apk/repositories
```
Comment (djdefi) [inline Dockerfile]: Same as the other bit, can remove this fully rather than comment it out if it no longer needed.

**Pull request merged**